packages/toolpad-components/src/Text.tsx

```diff
@@ -34,7 +42,25 @@ function Text({ value, markdown, href, loading, mode, sx, ...rest }: TextProps)
       ) : (
         <MarkdownContainer>
           <React.Suspense>
-            <ReactMarkdown remarkPlugins={[remarkGfm]}>{value}</ReactMarkdown>
+            <ReactMarkdown
+              options={{
+                overrides: {
+                  a: {
+                    component: MuiLink,
+                    props: {
+                      target: '_blank',
+                      rel: 'noopener noreferrer',
+                    },
+                  },
+                  pre: {
+                    component: CodeContainer,
+                  },
+                },
+                slugify: (text) => `toolpad-markdown-${text}`,
```

Comment:
But this would turn the links into `#toolpad-markdown-my-id`

Reply:
> When is id a problem exactly?

No idea. I left the feedback in case we rely on `getElementById`. If we don't, I think that keeping the id is simpler.